<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(87, 231, 146);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}

background-color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(178, 17, 17)
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(255, 77, 108);
}</string>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;

}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(136, 255, 243);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-bottom-right-radius: 30px;

}
QPushButton:pressed {
    background-color: rgb(136, 255, 243);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;

border-top-left-radius: 30px;

}
QPushButton:pressed {
    background-color: rgb(136, 255, 243);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;

}
QPushButton:pressed {
    background-color: rgb(136, 255, 243);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;

border-top-left-radius: 30px;

}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;



}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;

}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;

}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;

border-bottom-right-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(78, 214, 20);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rightBracket">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_leftBracket">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unar_minus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid grey;
 background-color:  transparent;
color: black;
border-top-right-radius: 30px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;
border-bottom-left-radius: 30px;
}
QPushButton:pressed {
    background-color: rgb(11, 213, 131)
}</string>
    </property>
    <property name="text">
     <string>X =</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
